include/contract_quda.h

```diff
@@ -5,5 +5,11 @@
 
 namespace quda
 {
+  // called in interface_quda.cpp, defined in contract.cu
+  void contractSummedQuda(const ColorSpinorField &x, const ColorSpinorField &y, std::vector<Complex> &result,
```

Comment:
Need to add doxygen here for this function.

Reply:
fixed with 28ffe81